zerver/data_import/slack.py

```
@@ -932,6 +933,24 @@ def channel_message_to_zerver_message(
         ]:
             continue
 
+        if "blocks" in message or "attachments" in message:
+            try:
+                # Don't crash the importer over rare cases where the message
+                # is formatted outside of what render_block and render_attachment
+                # expect.
+                formatted_block = process_slack_block_and_attachment(message)
+            except Exception:
+                print("Slack message has unexpected component format.")
```

Let's not eat the exception -- we want this to be debuggable.

---

I also removed this condition:
```python
      if "blocks" in message or "attachments" in message:
          try:
           ...
```
since all messages wil have the "blocks" components